e2e/test/scenarios/sharing/public-dashboard.cy.spec.js

```diff
@@ -102,14 +102,23 @@ describe("scenarios > public > dashboard", () => {
       .findByRole("switch")
       .check();
 
-    cy.wait("@publicLink");
-
-    cy.findByRole("heading", { name: "Public link" })
-      .parent()
-      .findByDisplayValue(/^http/)
-      .then($input => {
-        expect($input.val()).to.match(PUBLIC_DASHBOARD_REGEX);
-      });
+    cy.wait("@publicLink").then(({ response }) => {
+      expect(response.body.uuid).not.to.be.null;
+
+      cy.findByRole("heading", { name: "Public link" })
+        // This click doesn't have any meaning in the context of the correctness of this test!
```

Comment:
@nemanjaglumac just an idea from the top of my head - can't we wait for `role dialog` to disappear from DOM?

Reply:
It doesn't really disappear.
Actually, modal is a container for the div that has the transition. It translates from bottom to top and it goes from opacity 0 to 1.

The only thing we could wait for is for this div `to.have.css("top", "0px")` but I was afraid to go down that route because it's really an implementation detail that could change whenever, thus breaking the test.